controller/tap/server_test.go

```diff
@@ -240,7 +240,7 @@ status:
 
 			// TODO: mock out the underlying grpc tap events, rather than waiting an
 			// arbitrary time for request to timeout.
-			ctx, cancel := context.WithTimeout(context.Background(), 100*time.Millisecond)
+			ctx, cancel := context.WithTimeout(context.Background(), 200*time.Millisecond)
```

Comment:
intentional?

Reply:
Yea, that's for one of those tests where the tap request is asking for a pod that doesn't exist, so we're hitting Sync(), which takes a bit more of time. With less than 200ms we hit a timeout error.